modules/Students/student_view_details.php

```diff
@@ -504,8 +504,10 @@
 
                             $view = $_GET['view'] ?? 'grid';
                             if ($view == 'grid') {
-                                $table->setRenderer(new GridView($container->get('twig')));
-                                $table->getRenderer()->setCriteria($criteria);
+                                $table->setRenderer(
+                                    (new GridView($container->get('twig')))
```

Comment:
Thanks, well caught! Are we able to use the $container->get() method here to create the class, that way the constructor remains flexible and we can avoid using the new keyword to create classes.

Reply:
Rewritten.